[Modules] Format the metadata.json and aligned the Name and Description for all Modules and Updated Readme (#3309)
Azure/ResourceModules: modules/network/network-managers/network-groups/static-members/README.md

> # AVNM Network Group Static Members `[Microsoft.Network/networkManagers/networkGroups/staticMembers]`

Reviewer: ```suggestion
# Network Manager Network Group Static Members `[Microsoft.Network/networkManagers/networkGroups/staticMembers]`
```

> This module deploys an Azure Virtual Network Manager (AVNM) Network Group Static Member.

Reviewer: ```suggestion
This module deploys a Network Manager Network Group Static Member.
```
